### PR TITLE
Add Table of Contents, DG update for Find functionality

### PR DESCRIPTION
**Pull request by kynapy**

### CODE REVIEW

**Comment by kynapy** [comment]
Don't worry about the conflict guys, I'll resolve it

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -2,8 +2,32 @@
 layout: page
 title: Developer Guide
 ---
-* Table of Contents
-{:toc}
+# Developer Guide
+* Table of Contents {:toc}
```
Comment (TYKCodes) [inline docs/DeveloperGuide.md:6]: Great job on the neat formatting of the Table of Contents

**Review by TYKCodes — DISMISSED** [review]
Everything looks good to me
Thank you so much for updating the Table of Contents!

**Review by sprintaway — APPROVED** [review]
It is alright, but perhaps you want to refer to Parth's pull request of delete command because delete and find are both similar in format since they are split into 'user' and 'book' subcommand so you should follow Parth's DG formatting as well so that the DG is more consistent.

**Review by parth-io — APPROVED** [review]

**Pull request merged**